Makefile

```diff
@@ -14,15 +14,21 @@ gofmt:
 
 .PHONY: bin
 bin:
-	 go build -o ./out/chart-verifier main.go
+	cp -r ./config ./pkg/chartverifier/profiles
+	go build -o ./out/chart-verifier main.go
+	rm -rf ./pkg/chartverifier/profiles/config
 
 .PHONY: bin_win
 bin_win:
+	cp -r ./config ./pkg/chartverifier/profiles
 	env GOOS=windows GOARCH=amd64 go build -o .\out\chart-verifier.exe main.go
+	rm -rf ./pkg/chartverifier/profiles/config
```

Comment:
```suggestion
bin_win: copy-config
	env GOOS=windows GOARCH=amd64 go build -o .\out\chart-verifier.exe main.go
```

Reply:
decided to move profiles to a directory under pkg so this change is no longer needed.